frontend/src/metabase/components/DateAllOptionsWidget/DateAllOptionsWidget.tsx

```diff
@@ -1,19 +1,4 @@
-import { useState } from "react";
```

Comment:
I just moved the `DatePicker` part of this component to another file so we can add styles and props without affecting the current behavior of this component within dashboards